### ISSUE TITLE
Make File Explorer status bar transparent? (Windows 10) 

### ISSUE THREAD

**Issue by OlgaLeyla** [nptr/msstyleEditor#94]
Hello, 
I can change color, but I would like make it transparent. 
Could you please tell me Is it possible?
![ssssssss](https://user-images.githubusercontent.com/111338008/185997483-a01a2611-60dc-437e-b9ba-a0711eae8f2f.png)
![color](https://user-images.githubusercontent.com/111338008/185998064-ae1e53f5-f5c6-4725-9efb-331f6f8b4d19.JPG)

**Comment by nptr** [comment]
I don't know. If I had to guess I'd say its not possible. Transparency support was added retrospectively into Windows and was always an issue. There is at least no alpha channel to the colors and no alpha blending done with them afaik.

Even if you were to achieve transparency, I don't think it will look like expected. The status bar is not an overlay over the items view and image preview. The controls above just end there. So if the status bar were transparent (or not rendered without background) you'd probably see the default window color but not more of the above controls.
